.github/workflows/rav1e.yml

```diff
@@ -48,6 +48,7 @@ jobs:
     strategy:
       matrix:
         conf:
+         - build
```

Comment:
I think this is a great idea. However, could we have the run named something indicating this is for Rust beta (e.g. `beta-build`)?